assignments/array-methods.js

```diff
@@ -56,28 +56,87 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(function(runner) {
+    fullName.push(runner.first_name + ' ' + runner.last_name);
+});
 console.log(fullName);
 
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+
+runners.map((name) => {
+    allCaps.push(name.first_name.toUpperCase());
+});
```

Comment:
so this totally works and there's nothing wrong with it but I'd like to point out that since map returns an array every time you use it this would also work `allCaps = runners.map(runner => runner.first_name.toUpperCase())`
since map is making an array of what you return to it (or undefined if you return nothing) you can simply set allCaps equal to it's output

Reply:
looks better in one line one sec

Reply:
`allCaps = runners.map(runner => runner.first_name.toUpperCase())`

Reply:
like that